frontend/src/modules/accounts/hooks/useAccounts.tsx

```diff
@@ -0,0 +1,11 @@
+import { useContext } from 'react';
+import { useMutation } from 'react-query';
+import { ApiContext } from '../../../shared/providers/ApiProvider';
+
+export const useAccounts = () => {
+  const api = useContext(ApiContext);
+
+  const fetchUserById = useMutation((id: string) => api.getUserById(id));
+
+  return { fetchUserById };
```

Comment:
Used an object instead of the `fetchUserById` for I am thinking of expanding this implementation in future refactor.